development/extensions/tutorial_authentication.rst

```diff
@@ -109,12 +109,11 @@ authentication provider class is show below:
 The service file
 ++++++++++++++++
 For proper `dependency injection <https://wiki.phpbb.com/Dependency_Injection_Container>`_
-the provider must be added to ``services.yml``. The arguments are those
-of the provider's constructor and may be empty if no arguments are
+the provider must be added to ``services.yml``. The name of the service 
+must be in the form of auth.provider.<service name> in order for phpBB to register it.
```

Comment:
auth.provider.<service name> should be highlighted as inline code with double ``